frontends/web/src/routes/account/send/feetargets.tsx

```diff
@@ -58,8 +58,11 @@ class FeeTargets extends Component<Props, State> {
         options: null,
     };
 
+    private input = createRef<HTMLInputElement & {autofocus: boolean}>();
```

Comment:
Sorry for this 🙈  this was me trying to get autofocus working, I hope we will have less issues with React

Reply:
no problema!